### PR TITLE
build: change package scope clr -> cds

### PR DESCRIPTION
**Pull request by hippee-lee**
Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Type

What kind of change does this PR introduce?

Updates the build to publish `@cds/core, @cds/react`. This includes updating all core apps, the new website and Angular dev app. It also updates `@clr/city` dependencies to use `@cds/city`. 

Verification process includes:
1. Serving and building all core apps
2. serving and building the website app
3. serving and building the Angular dev app
4. publishing `@cds/core`, `@cds/react' and `@cds/city` to a dockerized Verdaccio proxy for npm
5. Creating an app with create-react-app that pulls packages from the local Verdaccio packages and setting up a simple app with CdsModal

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [x] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The build publishes all artifacts to the `@clr` scope and all apps with dependencies consume from the `@clr` scope. 

Issue Number: N/A

## What is the new behavior?
The build publishes only `core and react` artifacts to the `@cds` scope and all apps with dependencies consume those packages from the `@cds` scope. One additional change is to also consume `@cds/city` even though that is published from a different repo. 

## Does this PR introduce a breaking change?

- [x] Yes 
- [ ] No

Consumers will need to install core, react wrappers and clarity city font from the new scope:

```
yarn add @cds/core @cds/react @cds/city
```

The core apps depend on [this PR](https://github.com/vmware/clarity-city/pull/6).

**The build is failing b/c of yarn.lock. I'll need to update that after `@cds/city` is published on npm.**

### CODE REVIEW

**Comment by hippee-lee** [comment]
The build is failing b/c of yarn.lock. I'll need to update that after `@cds/city` is published on npm.

File: apps/core-angular-js/package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@clr/city": "^1.1.0",
-    "@clr/core": "./packages/core/dist/core",
+    "@cds/city": "^2.0.0",
```
Comment (coryrylan) [inline apps/core-angular-js/package.json]: Is 2.0 correct here?

Comment (hippee-lee) [reply apps/core-angular-js/package.json]: no, Clarity City is going to stay `1.1.0`. I'll push an update today.

Comment (hippee-lee) [reply apps/core-angular-js/package.json]: I pushed the change and will watch the build.

File: packages/schematics/src/update/update-3.0.0.ts [hunk 2]
```diff
@@ -42,8 +42,8 @@ export default function (options: any): Rule {
     // Add Clarity packages to package.json, if not found
     updateJsonFile('package.json', json => {
       const packages = Object.keys(json.dependencies);
-      if (!packages.includes('@clr/core')) {
-        json.dependencies['@clr/core'] = '^3.0.0';
+      if (!packages.includes('@cds/core')) {
```
Comment (gnomeontherun) [inline packages/schematics/src/update/update-3.0.0.ts]: Revert

Comment (hippee-lee) [reply packages/schematics/src/update/update-3.0.0.ts]: Done.

**Review by gnomeontherun — APPROVED** [review]

**Review by bdryanovski — APPROVED** [review]

**Comment by gnomeontherun** [comment]
I've rebased this commit and stacked the v5 beta on top in #5345.

**Pull request closed without merging**